remoulade/state/backend.py

```diff
@@ -17,29 +18,27 @@ class StateNamesEnum(Enum):
     Failure = "Failure"  # a `Message` that has been processed and raised an Exception
     Success = "Success"  # a `Message` that has been processed and does not raise an Exception
 
-    def __get__(self, instance, owner):
-        return self.value
-
 
 #: A type alias representing states in the database
-class State(namedtuple("State", ("name", "args", "kwargs"))):
+class State(namedtuple("State", ("message_id", "name", "args", "kwargs"))):
     """Catalog Class, it storages the state
         Parameters:
             name: Name of the state
             args: List of arguments in the state(name)
     """
 
-    def __new__(cls, name, args, kwargs):
-        if not any(state for state in StateNamesEnum if state.name == name):
+    def __new__(cls, message_id, name, args, kwargs):
+        if name not in list(StateNamesEnum):
```

Comment:
Why did you add list ?

Reply:
there was a warning, using non-Enums in containment checks will raise TypeError in Python 3.8